src/vimscript/exCommandParser.ts

```diff
@@ -575,7 +577,7 @@ export const builtinExCommands: ReadonlyArray<[[string, string], ArgParser | und
   [['v', 'global'], undefined],
   [['ve', 'rsion'], undefined],
   [['verb', 'ose'], undefined],
-  [['vert', 'ical'], undefined],
+  [['vert', 'ical'], VerticalCommand.argParser],
```

Comment:
```suggestion
  [['vert', 'ical'], VerticalCommand.argParser],
  [['vertical-','resize'], VerticalCommand.argParser],
```
only feedback I can provide is that `vertical-resize` is a valid command and it would be nice to support it.